pynest/examples/pong/run_simulations.py

```diff
@@ -0,0 +1,252 @@
+                self.game.reset_ball(False)
+                r_score += 1
+            elif self.game.result == pong.LEFT_SCORE:
+                self.game.reset_ball(True)
+                l_score += 1
+
+        end_time = time.time()
+        logging.info(
+            f"simulation of {max_runs} runs complete after: "
+            f"{datetime.timedelta(seconds=end_time-start_time)}")
+
+        self.game_data = np.array(self.game_data)
+
+        out_data = {}
+        out_data["ball_pos"] = self.game_data[:, 0]
+        out_data["left_paddle"] = self.game_data[:, 1]
+        out_data["right_paddle"] = self.game_data[:, 2]
+        out_data["score"] = self.game_data[:, 3]
+
+        logging.info("saving game data...")
+        with open(os.path.join(self.out_dir, "gamestate.pkl"), "wb") as file:
+            pickle.dump(out_data, file)
+
+        logging.info("saving network data...")
+
+        for net, filename in zip(
+            [self.player1, self.player2],
+                ["data_left.pkl.gz", "data_right.pkl.gz"]):
+            with gzip.open(os.path.join(self.out_dir, filename), "w") as file:
+                output = {"network_type": repr(net)}
+                output["with_noise"] = net.apply_noise
+                performance_data = net.get_performance_data()
+                output["rewards"] = performance_data[0]
+                output["weights"] = performance_data[1]
+                pickle.dump(output, file)
+
+        logging.info("done.")
+
+
+if __name__ == "__main__":
+    nest.set_verbosity("M_WARNING")
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--runs",
+                        type=int,
+                        default=5000,
+                        help="Number of game steps to simulate.")
+    parser.add_argument("--debug",
+                        action="store_true",
+                        help="Verbose debugging output.")
+    parser.add_argument("--out_dir",
+                        type=str,
+                        default="",
+                        help="Directory to save experiments to. Defaults to \
+                        current time stamp (YYYY-mm-dd-HH-MM-SS)")
+    parser.add_argument(
+        "--players", nargs=2, type=str, choices=["r", "rn", "d", "dn"],
+        default=["r", "rn"],
+        help="""types of networks that compete against each other. four
+    learning rule configuations are available:
+    r:  r-STDP without noise,
+    rn: r-STDP with noisy input,
+    d:  dopaminergic synapses without noise,
+    dn: dopaminergic synapses with noisy input.""")
+
+    args = parser.parse_args()
+
+    level = logging.DEBUG if args.debug else logging.INFO
+    format = '%(asctime)s - %(message)s'
+    datefmt = '%H:%M:%S'
+    logging.basicConfig(level=level, format=format, datefmt=datefmt)
+
+    p1, p2 = args.players
+    if p1[0] == p2[0] == 'd':
+        logging.error("""Nest currently (v3.1) does not support addressing
```

Comment:
Since I wrote this nothing has changed about the volume_transmitter, so this is still relevant. I feel like 'currently' should still be made explicit (e.g. by replacing the version with "August 2022"), or we could change the phrasing alltogether.